tests/unit/models/cart-model-test.js

```diff
@@ -93,6 +95,25 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it returns correct lineItemCount', function (assert) {
+  assert.expect();
```

Comment:
Yeah, expect should always specify a number. If assertions aren't run, we need to know.